pbs/pbsrequest.go

```diff
@@ -81,10 +87,12 @@ type PBSRequest struct {
 	IsDebug       bool            `json:"is_debug"`
 	App           *openrtb.App    `json:"app"`
 	Device        *openrtb.Device `json:"device"`
-	User          *openrtb.User   `json:"user"`
+	PBSUser       json.RawMessage `json:"user"`
+	SDK           *SDK            `json:"sdk"`
```

Comment:
is SDK supported on our `/openRTB2` endpoint?  Do we expect to send it in the future, or is it just for checking the version?